### ISSUE TITLE
May I ask about the  size of pedestrians used for training ? 

### ISSUE THREAD

**Issue by CyberPegasus** [sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection#22]
Since only pedestrians larger than 55 pixels are evaluated for MR metric following "Multispectral Pedestrian Detection: Benchmark Dataset and Baseline", I am wondering about the preprocessing details for your KAIST training set.
That is, in the training phase, only pedestrians larger than 55 pixels will be trained **or** pedestrians of all sizes will be trained?

**Comment by socome** [comment]
@CyberPegasus  The pedestrian larger than 12 pixels are trained and the other pedestrian are ignored by 'ignore_index' option of [torch.nn.CrossEntropyLoss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html). 

Please refer [code 1](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/8e583929a97b9f959634efa69174435ad0252a57/src/model.py#L651), [code 2](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/8e583929a97b9f959634efa69174435ad0252a57/src/datasets.py#L169) and [code 3](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/8e583929a97b9f959634efa69174435ad0252a57/src/config.py#L77).

**Comment by CyberPegasus** [comment]
Thanks for your reply ~
